mappings/net/minecraft/block/TorchFlowerBlock.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_ujhirzat net/minecraft/block/TorchFlowerBlock
```

Comment:
just a nitpick about capitalisation, since the identifier for this block is `minecraft:torchflower` I'd prefer we went with `TorchflowerBlock` as one word
make sure to add it to `minecraft_specific_words.txt`!